lib/modules/platform/azure/index.ts

```diff
@@ -745,7 +745,13 @@ export function massageMarkdown(input: string): string {
       'rename PR to start with "rebase!"'
     )
     .replace(regEx(`\n---\n\n.*?<!-- rebase-check -->.*?\n`), '')
-    .replace(regEx(/<!--renovate-(?:debug|config-hash):.*?-->/g), '');
+    .replace(regEx(/<!--renovate-(?:debug|config-hash):.*?-->/g), '')
+    .replace(
+      regEx(
+        /<!--[- ]*?renovate:start[- ]*?-->\s*|\s*<!--[- ]*?renovate:end[- ]*?-->/g
```

Comment:
don't allow the whitespace, if the tags are modified by users, ignore them

Reply:
Currently RenovateBot is the only source for applying those tags. But with this issue https://github.com/renovatebot/renovate/issues/17897 we will have `pr_templates.md` files which might contain those tags. This is why I wanted to keep pattern a little lose. 

```
Welcom to our repo, please add you comments ..
...
...

<!------------------ renovate:start ------------------------->
<!------------------ renovate:end -------------------------->

the rest of the stuff
```

Reply:
https://github.com/renovatebot/renovate/pull/19596/commits/4c8c82bc6923c4c8eb5a06df16038879c9ad413d